docs/quickstart-tenancy-and-apis.md

```diff
@@ -0,0 +1,314 @@
+  virtualWorkspaces:
+  - url: https://myhost:6443/services/apiexport/root:wildwest:cowboys-service/wildwest.dev
+```
+
+We can see that our `APIExport` has two key attributes in its status - its identity (more on this below) and a "virtual
+workspace" URL. You can think of this virtual workspace as behaving just like a workspace or cluster, except it searches
+across all true workspaces for instances of the resource types provided by the `APIExport`. We can use API discovery to
+see what resource types are available via this virtual workspace:
+
+```shell
+$ kubectl --server='https://myhost:6443/services/apiexport/root:wildwest:cowboys-service/wildwest.dev/clusters/*/' api-resources
+NAME      SHORTNAMES   APIVERSION              NAMESPACED   KIND
+cowboys                wildwest.dev/v1alpha1   true         Cowboy
+```
+
+The question is ... can we see the instance created by the consumer?
+
+```shell
+$ kubectl --server='https://myhost:6443/services/apiexport/root:wildwest:cowboys-service/wildwest.dev/clusters/*/' get -A cowboys \
+    -o custom-columns='WORKSPACE:.metadata.annotations.kcp\.dev/cluster,NAME:.metadata.name'
+WORKSPACE                     NAME
+root:wildwest:test-consumer   one
+```
+
+Yay!
+
+## APIs FAQ
+
+Q: Why do I have to append `/clusters/*/` to the `APIExport` virtual workspace URL?
+
+A: The URL represents the base path of a virtual kcp API server. With a standard kcp API server, workspaces live under
+the `/clusters/` path, so `/clusters/*/` represents a wildcard search across all workspaces via this virtual API
+server.
+
+Q: How should we understand an `APIExport` `identityHash`?
+
+A: Unlike with CRDs, a kcp instance might have many `APIResourceSchemas` of the same Group/Version/Kind, and users need
+some way of securely distinguishing them.
+
+Each `APIExport` is allocated a randomized private secret - this is currently just a large random number - and a public
+identity - just a SHA256 hash of the private secret - which securely identifies this `APIExport` from others.
+
+This is important because an `APIExport` makes a virtual workspace available to interact with all instances of a
+particular `APIResourceShema`, and we want to make sure that users are clear on which service provider `APIExports` they
+are trusting and only the owners of those `APIExport` have access to their resources via virtual workspaces.
+
+Q: Why do you have to use `--all-namespaces` with the apiexport virtual workspace?
+
+A: Think of this virtual workspace as representing a wildcard listing across all workspaces. It doesn't make sense to
+look at a specific namespace across all workspaces, so you have to list across all namespaces too.
+
+Q: What is reasoning behind the `APIBinding` "pointing to an ancestor or child of an ancestor" restriction? e.g. why
+can't I bind to the cowboys service in my home workspace?
+
+A: See https://github.com/kcp-dev/kcp/issues/2059
```

Comment:
This restriction has been removed in the main branch

Reply:
Yep, I've removed the FAQ on the "pointing to an ancestor or child of an ancestor" workspace restriction too.

Reply:
lol, nevermind - confusion - the other recent bugfix I incorporated was the single-character workspace name one